spec/requests/user_pages_spec.rb

```diff
@@ -56,4 +56,33 @@
 	  it { should have_title(user.name) }
 	end
 
+  describe "edit" do
+    let(:user) { FactoryGirl.create(:user) }
+    before { visit edit_user_path(user) }
+
+    describe "page" do
+      it { should have_content("Update your profile") }
+      it { should have_title("Edit user") }
+      it { should have_link("change", href: 'http://gravatar.com/emails') }
+    end
+
+    describe "with valid information" do
+      let(:new_name)  { "New Name" }
+      let(:new_email) { "new@example.com" }
+      before do
+        fill_in "Name",             with: new_name
+        fill_in "Email",            with: new_email
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.